sonar-project.properties

```diff
@@ -1,6 +1,13 @@
 sonar.projectKey=0xPolygonHermez_cdk-erigon
 sonar.organization=0xpolygonhermez
 
+sonar.exclusions=**/*_test.go,**/vendor/**,**/mocks/**,/test/**
+
+sonar.tests=.
+sonar.test.inclusions=**/*_test.go
+sonar.test.exclusions=**/vendor/**
+sonar.issue.enforceSemantic=true
+
 # Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
 sonar.sources=./zk,./zkevm
```

Comment:
Btw it is worth checking with Gateway folks do we need to include some other folders for scanning

(cc @hexoscott @mandrigin)

Reply:
We can add them in follow-up PRs